src/coreclr-debug/main.ts

```diff
@@ -1,263 +1,262 @@
-            return '.dylib';
-        case 'linux':
-            return '.so';
-        default:
-            throw Error('Unsupported platform ' + process.platform);
-    }
-}
-
-// Determines if the specified command is in one of the directories in the PATH environment variable.
-function isOnPath(command : string) : boolean {
-    let pathValue = process.env['PATH'];
-    if (!pathValue) {
-        return false;
-    }
-    let fileName = command;
-    let seperatorChar = ':';
-    if (process.platform == 'win32') {
-        // on Windows, add a '.exe', and the path is semi-colon seperatode
-        fileName = fileName + ".exe";
-        seperatorChar = ';';   
-    }
-    
-    let pathSegments: string[] = pathValue.split(seperatorChar);
-    for (let segment of pathSegments) {
-        if (segment.length === 0 || !path.isAbsolute(segment)) {
-            continue;
-        }
-        
-        const segmentPath = path.join(segment, fileName);
-        if (existsSync(segmentPath)) {
-            return true;
-        }
-    }
-    
-    return false;
-}
-
-function ensureAd7EngineExists(channel: vscode.OutputChannel, outputDirectory: string) : Promise<void> {
-    let filePath = path.join(outputDirectory, "coreclr.ad7Engine.json");
-    return new Promise<void>((resolve, reject) => {
-        fs.exists(filePath, (exists) => {
-            if (exists) {
-                return resolve();
-            } else {
-                channel.appendLine(`${filePath} does not exist.`);
-                channel.appendLine('');
-                // NOTE: The minimum build number is actually less than 1584, but this is the minimum
-                // build that I have tested.
-                channel.appendLine("Error: The .NET CLI did not correctly restore debugger files. Ensure that you have .NET CLI version 1.0.0 build #001584 or newer. You can check your .NET CLI version using 'dotnet --version'.");
-                return reject("The .NET CLI did not correctly restore debugger files.");
-            }
-        });
-    });
-}
-
-function spawnChildProcess(process: string, args: string[], channel: vscode.OutputChannel, workingDirectory: string) : Promise<void> {
-    const promise = new Promise<void>(function(resolve, reject) {
-        const child = child_process.spawn(process, args, {cwd: workingDirectory});
-
-        child.stdout.on('data', (data) => {
-            channel.append(`${data}`);
-        });
-
-        child.stderr.on('data', (data) => {
-            channel.appendLine(`Error: ${data}`);
-        });
-
-        child.on('close', (code: number) => {
-            if (code != 0) {
-                channel.appendLine(`${process} exited with error code ${code}`);
-                reject(new Error(code.toString()));    
-            }
-            else {
-                resolve();
-            }
-        });
-    });
-
-    return promise;
+/*---------------------------------------------------------------------------------------------
```

Comment:
not sure why this file is showing as one big delete and add. possibly because of the rename? I thought github could handle this better. Maybe I'm missing something? Doesn't seem to be a line ending thing.